packages/@dcl/inspector/src/lib/logic/catalog.ts

```diff
@@ -54,9 +54,15 @@ export function getAssetsByCategory(assets: Asset[]) {
   return categories
 }
 
-export function isSmart(asset: Asset) {
-  const components = Object.keys(asset.components)
-  return components.length > 1 || (components.length === 1 && components[0] !== CoreComponents.GLTF_CONTAINER)
+export function isSmart(asset: Partial<Asset>) {
+  const components = Object.keys(asset?.components ?? {})
+  return (
+    (components.length > 1 &&
+      components.some((component) =>
+        [ComponentName.ACTIONS, ComponentName.TRIGGERS].find(($) => $.toString() === component)
+      )) ||
+    (components.length === 1 && components[0] !== CoreComponents.GLTF_CONTAINER)
```

Comment:
i dont get this logic. If components.length === 1 and its not a gltf container its a smart item ?
what if you create an empty entity with a transform ? (sometimes they use this to agroup entities)